testsuite/fuzzer/fuzz.sh

```diff
@@ -241,6 +284,16 @@ function coverage() {
         -Xdemangler=rustfilt \
         --show-branches=count \
         --ignore-filename-regex='rustc/.*/library|\.cargo'
+
+
+    cargo +$NIGHTLY_VERSION cov -- show $fuzz_target_bin \
+        --format=text \
+        --instr-profile=fuzz/coverage/$fuzz_target/coverage.profdata \
+        --show-directory-coverage \
+        --output-dir=$text_coverage_dir \
+        -Xdemangler=rustfilt \
+        --show-branches=count \
+        --ignore-filename-regex='rustc/.*/library|\.cargo'
```

Comment:
Why are you calling two time the coverage generator? If you need it with different flags, parametrize the invocation.